docs/stage-1/README.md

```diff
@@ -1,6 +1,25 @@
-# sEURO Offering Architecture
-The sEURO Offering is the first stage in the IBCO process. Users can obtain sEURO in exchange for ETH or some accepted ERC20 tokens, at a discount which gradually reaches full price during the period of the IBCO.
+# Stage 1
+
+The sEURO offering is the first stage in the IBCO process. Users can obtain sEURO in exchange for ETH or some accepted ERC20 tokens, at a discount which gradually reaches full price during the period of the IBCO.
+
+### sEURO Offering public APIs
+
+**readOnlyCalculatePrice(uint256 _euroAmount)**
+*Description*: reads the cached price of sEUROs from the bonding curve
+*Input*: an amount of FIAT euros
+*Output*: an estimate in sEUROs given the input
 
+**calculatePrice(uint256 _euroAmount)**
+*Description*: calculates the actual (current) price of sEUROs in the bonding curve
+*Input*: an amount of FIAT euros
+*Output*: actual amount of sEUROs given the input
+
+**updateCurrentBucket(uint256 _minted)**
+*Description*: updates the current point on the bucket based on the IBCO total supply
+*Input*: an amount of sEUROs to add to the IBCO total supply
```

Comment:
this is the interface for the **BondingCurve**, not the **SEuroOffering** (and therefore the public ones)

Reply:
i mean the `SEuroOffering` contract is the most public one. shouldn't that be the one that is described in the docs here?